### ISSUE TITLE
We should have min/max target estimates of problems

### ISSUE THREAD

**Issue by nojhan** [IOHprofiler/IOHexperimenter#58]
On a separate accessors than optimum.

**Comment by nojhan** [comment]
This would be useful to automatically derivate ECDF logger's target bounds for a problem/suite.
If the information is not available, then accessors would raise an error and the user would defaults to hand-picked bounds.

Then additional simple constructors would be interesting, like:
```cpp
IOHprofiler_ECDF_logger(IOHprofiler_Problem& pb, double max_evals, size_t buckets);
// derivates target bounds from pb and use square domain.
```

**Comment by nojhan** [comment]
Is it closed because it's done or close because it will not be added?

**Comment by wangronin** [comment]
I think because is it done (not super sure..)

@jacobdenobel did you incorporate this one in the new code structure? I don' recall anything..

**Comment by jacobdenobel** [comment]
This was closed because #54 was merged, in a comment there it was mentioned that this also included a feat for fixing this issue. 

This is sligthly modified in the restructured version, and there is no seperate accessor for ```has_opt```, but you can infer this information like so: ```!problem.state().objective.y.empty()```. I think this settles this issue. If not, please let me know.

**Comment by nojhan** [comment]
Thanks! (and always put a comment when closing issues, that way I will not bother you anymore ;-)
